backend/editor/api.py

```diff
@@ -25,6 +27,26 @@
     expose_headers=["*"],
 )
 
+@app.on_event("startup")
+async def startup():
+    """
+    Initialize Neo4J database
+    """
+    global driver, session
+    uri = "bolt://localhost:7687"
+    driver = GraphDatabase.driver(uri)
+    session = driver.session()

```

Comment:
well done. 

Could we take the uri from environment and put this env reading in a settings.py file (just to group them, also because it's easier to override them for test)

That is you will have setting.py with

```
uri = os.environ.get("NEO4J_URI", "bolt://localhost:7687")
```

and in api.py:

```
import settings

uri = settings.uri
```

(it's intentional not to write `from settings import uri` but `import settings` to ease monkeypatching it for tests, if needed)